docs/swagger.yaml

```diff
@@ -0,0 +1,405 @@
+
+  MsgCreateDid:
+    type: object
+    required:
+      - payload
+      - signatures
+    properties:
+      payload:
+        $ref: '#/definitions/MsgCreateDidPayload'
+      signatures:
+        description: Payload should be signed by all `controller` private keys. The `signatures` must contains signatures from all controllers.
+        type: array
+        items:
+          $ref: '#/definitions/SignInfo'
+
+  MsgCreateDidPayload:
+    required:
+      - id
+    properties:
+      id:
+        description: Target DID as base58-encoded string for 16 or 32 byte DID value with cheqd DID Method prefix `did:cheqd:<namespace>:`.
+        type: string
+        example: did:cheqd:testnet:6cgbu8ZPoWTnR5Rv5JcSMB
+      verificationMethod:
+        description: A list of Verification Methods
+        type: array
+        items:
+          $ref: '#/definitions/VerificationMethod'
+      controller:
+        description: A list of strings with key aliases or IDs
+        type: array
+        items:
+          type: string
+          example: did:cheqd:mainnet:N22KY2Dyvmuu2PyyqSFKue
+      authentication:
+        description: A list of strings with key aliases or IDs
+        type: array
+        items:
+          type: string
+          example: "did:cheqd:mainnet:N22KY2Dyvmuu2PyyqSFKue#key1"
+      assertionMethod:
+        description: A list of strings with key aliases or IDs
+        type: array
+        items:
+          type: string
+          example: "did:cheqd:mainnet:N22KY2Dyvmuu2PyyqSFKue#key1"
+      capabilityInvocation:
+        description: A list of strings with key aliases or IDs
+        type: array
+        items:
+          type: string
+          example: "did:cheqd:mainnet:N22KY2Dyvmuu2PyyqSFKue#key1"
+      capabilityDelegation:
+        description: A list of strings with key aliases or IDs
+        type: array
+        items:
+          type: string
+          example: "did:cheqd:mainnet:N22KY2Dyvmuu2PyyqSFKue#key1"
+      keyAgreement:
+        description: A list of strings with key aliases or IDs
+        type: array
+        items:
+          type: string
+          example: "did:cheqd:mainnet:N22KY2Dyvmuu2PyyqSFKue#key1"
+      alsoKnownAs:
+        description: A list of strings. A DID subject can have multiple identifiers for different purposes, or at different times. The assertion that two or more DIDs refer to the same DID subject can be made using the `alsoKnownAs` property.
+        type: array
+        items:
+          type: string
+      service:
+        description: A set of Service Endpoint maps
+        type: array
+        items:
+          $ref: '#/definitions/Service'
+      context:
+        description: A list of strings with links or JSONs for describing specifications that this DID Document is following to.
+        type: array
+        items:
+          type: string
+          example: "https://www.w3.org/ns/did/v1"
```

Comment:
Not sure if this is a Swagger doc issue only or in the actual code: the `context` section of a DID Document is right at the top, before everything else, and starts with `@context` before the block begins. If that's what we're doing, then this is a simple cosmetic fix in this file, otherwise we might need to check our implementation.

Reply:
Unfortunately, the field name in protobufs cannot start with `@`, but I think we can add it in DIDDoc resolver in JSON converter for passing the test suites.
Regarding fields order, it is not strictly defined in JSON, only in protobuff. In the proto and documentation, it is located at the end, since I followed the logic of specifying the required fields first, such as ID. Nevertheless, I wanted to separate the context from the other fields, so it was moved to the end.
In fact, we can move `context` to the top, if necessary.